.travis.yml

```diff
@@ -19,5 +16,12 @@ install:
 before_script:
   - export PYTHONPATH=$PYTHONPATH:$(pwd)
 
-script:
-  - (cd testing && make test_all)
+matrix:
+  include:
+    - python: 2.7
+    script:
+      - (cd testing && make test_all)
+
+    - python: 3.9
+    script:
```

Comment:
should we make the python3 as the primary script?

Reply:
The modules that work within the boptest docker are Python 2.7 and are tested within a 2.7 environment (/data, /kpis, /parser).  The code that would utilize Python 3 if it is used for the main testing script is the example controllers and the unit testing code for the example controllers and the test cases.  The unit testing code could be updated to also work with Python 3, as in https://github.com/ibpsa/project1-boptest/pull/229/commits/29d664cd5d088b823bdc2dc6a0f70fc0f00db11c.  I think the only consequence would be the need to update reference results since there could be some numeric differences with Python 3 versions of pandas, numpy, etc.

I've made the updates to the unit testing code and am testing now to see if this is true.

Reply:
Indeed, reference results for timeseries just need to be updated for testcase1 and testcase3 for small numeric differences.  And two results in bestest_hydronic_heat_pump need to be updated due to python3 seemingly to order a list differently in two cases.

Reply:
Ok per @JavierArroyoBastida comment below and discussions with @SenHuang19 and @kbenne, will run ``test_all`` in python 3 environment, requiring numeric updates to reference results as described above, and run example controllers in python only with python 2 to maintain compatibility for now.  Basically switch what is in the PR right now.

Reply:
Done in https://github.com/ibpsa/project1-boptest/pull/336/commits/2a4af129e48fc9c990cf86ed54bf939fa2a67d10.